modules/manage/partials/remote-read-replicas.adoc

```diff
@@ -58,9 +57,7 @@ You must configure access to the same object storage as the origin cluster.
 ifndef::env-kubernetes[]
 To set up a Remote Read Replica topic on a separate remote cluster:
 
-. Create a remote cluster for the Remote Read Replica topic.
-* If that's a multi-region bucket/container, you can create the read replica cluster in any region that has that bucket/container.
-* If that's a single-region bucket/container, the remote cluster must be in the same region as the bucket/container.
+. Create a remote cluster for the Remote Read Replica topic. The remote cluster can be in the same or a different region as the bucket/container.
```

Comment:
Should you have the same info here in the step; i.e., for AWS, the remote cluster must be in the same region as the origin cluster, but for GCP, the remote cluster can be in the same or a different region as the origin cluster.